### ISSUE TITLE
is this package not worked on yarn v1.0.x versions

### ISSUE THREAD

**Issue by chilijung** [AndersDJohnson/use-yarn#2]
Before v1.0.x version this package worked normal.  After upgraded to v1.0.x `use-yarn` can't detect I'm using yarn.

```
➜  projects git:(master) yarn
yarn install v1.0.2
$ use-yarn || yarn global add use-yarn
This project uses yarn instead of npm. Please upgrade!
See https://yarnpkg.com.

To install dependencies:
$ yarn install

To install a new package (also adds to package.json):
$ yarn add [packagename]

To upgrade a package:
$ yarn upgrade [packagename]
```

**Comment by chilijung** [comment]
After some research, https://github.com/yarnpkg/yarn/pull/4330 after yarn v1.0.1 version `npm_lifecycle_script` is added in `process.env`, so I think this package is broken?

**Comment by chilijung** [comment]
https://github.com/AndersDJohnson/is-npm-not-yarn/pull/1 , this will fix this issue.

**Comment by AndersDJohnson** [comment]
@chilijung Thanks. Updated and published as `2.0.0`.
